exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -557,7 +556,7 @@ private void readBlockMeta(String path,
     mapper.registerModule(serialModule);
     mapper.registerModule(module);
     mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-    FSDataInputStream is = fs.open(p);
+    FSDataInputStream is = fs.open(path);
```

Comment:
Common practice is to enclose this in a try-with-resources block, assuming that this stream derives from `AutoCloseable`. Else, we should add our own `try` block and close the file in `finally`.